angular.json

```diff
@@ -52,7 +57,7 @@
                         "main": "projects/demo/src/main.ts",
                         "polyfills": "projects/demo/src/polyfills.ts",
                         "tsConfig": "projects/demo/tsconfig.app.json",
-                        "aot": false,
+                        "aot": true,
```

Comment:
@alexsukhodolsky  I would be happy to merge it but unfortunately the unit test compilation is failing. Probably due to this change. @lehoffma Can you check this please?

Reply:
I tried that, but it didn't change the test results for me. The broken unit tests were mostly due to the updated libraries like rxjs I think. I rebased and pushed a fix for the tests, let me know if there's anything else I can do.